src/Entity/ServiceProvider.php

```diff
@@ -106,6 +106,20 @@ class ServiceProvider extends AbstractRole
     public $requestedAttributes;
 
     /**
+     * @var null|string
+     *
+     * @ORM\Column(name="url_en", type="string")
+     */
+    public $urlEn;
```

Comment:
These are used to define support URLs. Should we allow Janus to dictate the naming here? Wouldn't it be better to actually name these `supportUrlEn` and `supportUrlNl`? @relaxnow?